src/Arcus.Observability.Telemetry.Core/Logging/RequestLogEntry.cs

```diff
@@ -46,6 +80,7 @@ public class RequestLogEntry
             RequestUri = uri;
             ResponseStatusCode = statusCode;
             RequestDuration = duration;
+            OperationName = operationName;
```

Comment:
We should guard not null or whitespace here

Reply:
Added, see: https://github.com/arcus-azure/arcus.observability/pull/229/commits/4fb6cf03292fa42a0876bfc0d1ba5db4716c3554